app/_hub/kong-inc/http-log/index.md

```diff
@@ -57,41 +57,57 @@ params:
       required: true
       default:
       value_in_examples: http://mockbin.org/bin/:id
+      datatype: Reviewers cannot locate in typedefs schema, where is typedefs.url schema?
       description: The HTTP endpoint (including the protocol to use) to which the data is sent.
     - name: method
       required: false
       default: "`POST`"
       value_in_examples: POST
+      datatype: string
       description: |
-        An optional method used to send data to the HTTP server. Supported values are `POST` (default), `PUT`, and `PATCH`.
+        An optional method used to send data to the HTTP server. Supported values are
+        `POST` (default), `PUT`, and `PATCH`.
+    - name: content_type
+      required: false
+      default: "`application/json`"
+      value_in_examples:
+      datatype: string
+      description: |
+      REVIEWERS need description. The only available option is `application/json`.
     - name: timeout
       required: false
       default: "`10000`"
       value_in_examples: 1000
+      datatype: number
       description: An optional timeout in milliseconds when sending data to the upstream server.
     - name: keepalive
       required: false
       default: "`60000`"
       value_in_examples: 1000
+      datatype: number
       description: An optional value in milliseconds that defines how long an idle connection will live before being closed.
     - name: flush_timeout
       required: false
       default: "`2`"
       value_in_examples: 2
+      datatype: number
       description: |  
         Optional time in seconds. If `queue_size` > 1, this is the max idle time before sending a log with less than `queue_size` records.    
     - name: retry_count
       required: false
       default: 10
       value_in_examples: 15
+      datatype: integer
       description: Number of times to retry when sending data to the upstream server.
     - name: queue_size
       required: false
       default: 1
+      datatype: integer
       description: Max number of log entries to be sent on each message to the upstream server.
     - name: headers
       required: false
       default: empty table
+      datatype: array of string elements
       description: An optional table of headers added to the HTTP message to the upstream server.
```

Comment:
In the description, I'd suggest we communicate to users that some headers are not allowed: https://github.com/Kong/kong/blob/master/kong/plugins/http-log/schema.lua#L24-L33

They are: `Host`, `Content-Length`, `Content-Type`

Reply:
Done in commit f298b1a. The plugin will probably display on hub with this build. Spacing was off for content type.